### ISSUE TITLE
(VoskAPI:UpdateGrammarFst():recognizer.cc:351) Ignoring word missing in vocabulary

### ISSUE THREAD

**Issue by niuhaitaofeilong** [alphacep/vosk-api#2045]
Ubuntu18.04   model-small-chinese-0.22
std::string chinese_hotwords_grammr = R"(["北京科技馆"])";
vosk_recognizer_set_grm(recognizer, chinese_hotwords_grammr.c_str());

Report the log as follows,

LOG (VoskAPI:ReadDataFiles():model.cc:223) Decoding params beam=12 max-active=5000 lattice-beam=4
LOG (VoskAPI:ReadDataFiles():model.cc:226) Silence phones 1:2:3:4:5:6:7:8:9:10
LOG (VoskAPI:RemoveOrphanNodes():nnet-nnet.cc:948) Removed 0 orphan nodes.
LOG (VoskAPI:RemoveOrphanComponents():nnet-nnet.cc:847) Removing 0 orphan components.
LOG (VoskAPI:ReadDataFiles():model.cc:259) Loading i-vector extractor from data/vendor/cmn_file/cmn_model-small-chinese-0.22/ivector/final.ie
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:183) Computing derived variables for iVector extractor
LOG (VoskAPI:ComputeDerivedVars():ivector-extractor.cc:204) Done.
LOG (VoskAPI:ReadDataFiles():model.cc:295) Loading HCL and G from data/vendor/cmn_file/cmn_model-small-chinese-0.22/graph/HCLr.fst data/vendor/cmn_file/cmn_model-small-chinese-0.22/graph/Gr.fst
LOG (VoskAPI:ReadDataFiles():model.cc:321) Loading winfo data/vendor/cmn_file/cmn_model-small-chinese-0.22/graph/phones/word_boundary.int
WARNING (VoskAPI:Check():nnet-nnet.cc:789) Node tdnn6.relu is never used to compute any output.
WARNING (VoskAPI:Check():nnet-nnet.cc:789) Node tdnn6.batchnorm is never used to compute any output.
WARNING (VoskAPI:Check():nnet-nnet.cc:789) Node output.affine is never used to compute any output.
WARNING (VoskAPI:Check():nnet-nnet.cc:789) Node output.log-softmax is never used to compute any output.
LOG (VoskAPI:RemoveOrphanNodes():nnet-nnet.cc:948) Removed 4 orphan nodes.
LOG (VoskAPI:RemoveOrphanComponents():nnet-nnet.cc:847) Removing 4 orphan components.
LOG (VoskAPI:Collapse():nnet-utils.cc:1488) Added 0 components, removed 4
LOG (VoskAPI:UpdateGrammarFst():recognizer.cc:330) ["Ƽ"]
WARNING (VoskAPI:UpdateGrammarFst():recognizer.cc:351) Ignoring word missing in vocabulary: 'Ƽ'
LOG (VoskAPI:Estimate():language_model.cc:140) Estimating language model with ngram-order=2, discount=0.5
LOG (VoskAPI:OutputToFst():language_model.cc:207) Created language model with 1 states and 0 arcs.
LOG (VoskAPI:OutputToFst():language_model.cc:209) Originally language model with 1 states and 0 arcs.
Recognizer model loaded successfully.
Speaker model loaded successfully.
1 test_voice_Chinese/beijingkejiguan.wav
final: {
  "text" : ""
}
****************************************************************************************************************
2
Segmentation fault
